README.md

```diff
@@ -21,7 +21,7 @@ This repository also contains the Zowe Node Client SDK. The SDK lets you leverag
 <br/>
 
 ## **Documentation**
-For detailed information about how to install, configure, and use Zowe CLI, see [Zowe CLI Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/). The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.
+For detailed information about how to install, configure, and use Zowe CLI, see [Zowe CLI Quickstart Documentation](https://docs.zowe.org/stable/getting-started/cli-getting-started/) or the more thorough [Zowe CLI Documentation](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/). The documentation includes examples and tutorials for how to contribute to Zowe CLI and develop CLI plug-ins.
```

Comment:
Should we make the same change to `packages/cli/README.md`? People only see the top-level one when they browse our repo on GitHub: https://github.com/zowe/zowe-cli. The readme inside `packages/cli` is the one shown on npm: https://www.npmjs.com/package/@zowe/cli

Reply:
Yes whoops